### DISCUSSION TITLE
build ebpf for kernel 5.19 in arch linux

### DISCUSSION THREAD

**Discussion by Maziar123** [evilsocket/opensnitch#730] [Q&A] [answered]
Hi
The **kernel 5.19** is now stable for arch
the AUR package for build for opensnitch-ebpf-module-git is out of date

Also, build wiki for ebpf is outdated …

Can any help for active epbf in arch Linux  ?

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @Maziar123 , the last time I compiled opensnitch-ebpf-module-git it downloaded latest changes from git. 

Take a look at these 2 comments: https://github.com/evilsocket/opensnitch/issues/707#issuecomment-1192963933 and https://github.com/evilsocket/opensnitch/issues/707#issuecomment-1200224815
